_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+
+  * Workshop at PEARC20: We've had a workshop entitled "[Research Software Engineers Community Workshop](https://us-rse.org/events/2020/2020-07-pearc20)" accepted as part of the PEARC20 virtual conference this year. The workshop will provide an opportunity for research software engineers to connect, share their work, and contribute to the growth of the community. There will be presentations from RSEs about their work and their groups, and about the development of the RSE community. We will work together to identify key tasks for building the research software engineering community and organize working groups to address these issues after the conference.
+ 
+  * Events at SC20: A half-day workshop has been accepted, entitled "Research Software Engineers in HPC:  Creating Community, Building Careers, Addressing Challenges (RSE-HPC-2020)". We also expect additional RSE-related events to be part of the SC20 program as it further develops.
+
+**RSE Stories**. The [Research Software Engineer Stories (RSE Stories) podcast](https://us-rse.org/rse-stories) was founded in September 2019 by RSE Vanessa Sochat in an effort to better share the diverse kinds of life experiences and roles held by real research software engineers. As of April 2020 the podcast has released 15 episodes that feature RSEs internationally, ranging from open source developers, to academic positions, to high performance computing staff. The podcast is available on iTunes and most subscription services, and has been [shared at Stanford](https://itcommunity.stanford.edu/news/research-software-engineer-rse-podcast-builds-it-community), with the UK-RSE community, and episodes will continue to be posted on the US-RSE Twitter.
+
+**Syndicated Blogs**. The US-RSE [syndicated blog](https://us-rse.org/blog) is a GitHub site that, as of April 2020, parses feeds of 9 well-known RSE groups and individual research software engineers. The blog is updated nightly in an automated fashion to re-parse feeds, and update the site content with new content from any of the input feeds. The blog provides a single interface for someone interested in research software engineering to read a diverse set of opinions, and associated work. Research software engineers or groups that have not added their feed are encouraged to do so at [the repository](https://github.com/usrse/blog).
+
+**Funding agency interest**. We've had informal interest in the topic of RSEs from a number of public funding agencies and private foundations. One example of such interest turning into funding is from the Alfred P. Sloan Foundation, which has provided an award to support our community building activities, including travel funding for a limited number of participants to the planned Princeton workshop as well as  part-time support for a community coordinator, who, in addition to organizing the workshop, will become more involved in US-RSE community events in the coming months.. 
```

Comment:
This is great! Part of this conversation should be something along the lines of "How can RSEs from different institutions better work together and learn from one another?" This is something I keep bringing up to my group / at my institution and it's a hard question, but important I think.

Reply:
I completely agree with you, but don't want to add anything in this post about this issue

Reply:
@vsoch - ok to mark this part as resolved?